googletest/src/gtest-filepath.cc

```diff
@@ -236,7 +236,7 @@ bool FilePath::DirectoryExists() const {
     result = true;
   }
 #else
-  posix::StatStruct file_stat;
+  posix::StatStruct file_stat = {};
```

Comment:
This change feels unrelated to the rest of what's going on here.  Is this needed to make esp8266 work, or was this just a drive-by cleanup?

Reply:
I think this was drive by clean up.  I can revert the line if desired.